### ISSUE TITLE
Angular 9 and Ivy Compatibility

### ISSUE THREAD

**Issue by b-johnse** [akveo/nebular#2040]
### Issue type

**I'm submitting a ...**  (check one with "x")

* [x] bug report
* [ ] feature request

### Issue description

**Current behavior:**

When opt-in for Angular 9 and the new ivy compiler an error is given:
```
ERROR in ../../node_modules/@nebular/theme/components/datepicker/datepicker.component.d.ts:176:40 - error TS2314: Generic type 'NbBasePicker<D, T, P>' requires 3 type argument(s).
```

**Expected behavior:**

Compile without issues

**Steps to reproduce:**

On any Angular 8 app using nebular:
- ng update @angular/core@next 
- (wait for update schematic to finish)
- ng serve, and error appears

**Related Code**
In node_modules/@nebular/theme/components/datepicker/datepicker.component.d.ts,  
 NbBasePicker<D, T, P> is transformed via the new `"postinstall": "ngcc --properties es2015 browser module main --first-only --create-ivy-entry-points",`
```
static ngBaseDef: ɵngcc0.ɵɵBaseDef<NbBasePicker>;
```

### Other information:

**npm, node, OS, Browser**
```
npm: 6.10.0
node: 12.7.0
```

**Angular, Nebular**
```
Angular: 9.0.0-next-13
Nebular: 4.3.1
```

**Comment by rayman1104** [comment]
Looks like this is fixed since 9.0.0-next-14.
https://github.com/angular/angular/pull/33264

**Comment by yggg** [comment]
Duplicate of #1946. Closing as resolved. Thanks @rayman1104!
